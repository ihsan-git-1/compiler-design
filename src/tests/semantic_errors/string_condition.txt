class test {

    int a(){
            int b =3;
            String a="test";
                if(1 > a){
                    int count = 3;

        }
    }
}