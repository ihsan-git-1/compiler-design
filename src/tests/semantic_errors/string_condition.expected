String Can't Be Compared To Expression
